build.gradle

```diff
@@ -129,12 +129,12 @@ allprojects {
   }
 
   task sourcesJar(type: Jar, dependsOn: classes) {
-    classifier = 'sources'
+    archiveClassifier = 'sources'
```

Comment:
Why did this need to change?

Reply:
These were warnings in build.gradle file that will be an issue in future release of gradle. Although they should not be technically part of this PR, I fixed them originally while researching this problem and were using debug flags to the build.